dataland-frontend/src/utils/GenericFrameworkTypes.ts

```diff
@@ -15,16 +15,18 @@ export interface Subcategory {
 }
 
 export interface Field {
-  showIf: (dataModel?: LksgData | PathwaysToParisData) => boolean;
+  showIf: (dataModel?: LksgData | PathwaysToParisData | SfdrData) => boolean;
   name: string;
   label: string;
   description: string;
+  unit?: string;
   component: string;
   dependency?: string;
   certificateRequiredIfYes?: boolean;
   validation?: string;
   validationLabel?: string;
   required?: boolean;
+  evidenceDesired?: boolean;
```

Comment:
See my other comment where I asked why we need this since it is always set to the same values.

Reply:
Maybe discuss this in the call if we need this prop or not.

Reply:
@stewal19  Yes let's talk about it.   I will look a little deeper into the code and then hit you up to discuss this later.

Reply:
I am able to discuss it now.